### DISCUSSION TITLE
No new results in Tpot UI after modifying tpot.yml

### DISCUSSION THREAD

**Discussion by Gornlord** [telekom-security/tpotce#762] [Q&A]
Hi everyone, big thank you for this project, its been a great learning experience for me so far.
I've been following the guide(https://github.com/telekom-security/tpotce/wiki/Reconfigure-logstash.conf) to get tpot results to output via syslog and I'm running into an issue where the moment I add this line: "- /data/elk/logstash.conf:/etc/logstash/conf.d/logstash.conf" I no longer see any new results in the tpot UI. Am I doing something wrong?

**Comment by RogerWeihrauch** [comment]
Hi.
You have access/opened to/ port 1903?
And also a 'user' w/ access rights:
if [type] == "Suricata" {         
      file {                             
        file_mode => **0760**                                                                                                                                                                                                                        
        path => "/data/suricata/log/suricata_ews.log"
Did you also try the part w/ debug output following this?
May I misunderstood you in other case.
